gmail/Package.md

```diff
@@ -1,79 +1,112 @@
+**Message Operations**
 
-3. Create a new Ballerina project by executing the following command.
+The `wso2/gmail` package contains operations to send emails in Text and HTML formats with attachments and inline images. 
+It supports searching and reading messages in Gmail using Gmail filters. The package also supports trashing, untrashing, 
+and deleting messages as well.
 
-    ```shell
-    $ ballerina init
-    ```     
+**Thread Operations**
 
-4. Import the Gmail package to your Ballerina program as follows.
+The `wso2/gmail` package contains operations to read, search, trash, untrash, and delete mail threads in Gmail.
 
-    ```ballerina
-    import ballerina/io;
-    import wso2/gmail;
+**UserProfile Operations**
 
-    //User credentials to access Gmail API
-    string accessToken = "<access_token>";
-    string clientId = "<client_id>";
-    string clientSecret = "<client_secret>";
-    string refreshToken = "<refresh_token>";
-    //The user's email address. The special value "me" can be used to indicate the authenticated user.
-    string userId = "me";
+The `wso2/gmail` package contains operations to get Gmail user profile details.
 
-    function main(string... args) {
-
-       endpoint gmail:Client gmailEP {
-           clientConfig:{
-               auth:{
-                   accessToken:accessToken,
-                   clientId:clientId,
-                   clientSecret:clientSecret,
-                   refreshToken:refreshToken
-               }
-           }
-       };
-
-       //Set headers, message body and etc to MessageRequest.
-       gmail:MessageRequest messageRequest;
-       messageRequest.recipient = "recipient@mail.com";
-       messageRequest.sender = "sender@mail.com";
-       messageRequest.cc = "cc@mail.com";
-       messageRequest.subject = "Email-Subject";
-       messageRequest.messageBody = "Email Message Body Text";
-       //Set the content type of the mail as TEXT_PLAIN or TEXT_HTML.
-       messageRequest.contentType = gmail:TEXT_PLAIN;
-
-       //Call the Gmail endpoint function sendMessage().
-       var sendMessageResponse = gmailEP -> sendMessage(userId, messageRequest);
-       match sendMessageResponse {
-           (string, string) sendStatus => {
-               //For a successful message request, returns message and thread id.
-               string messageId;
-               string threadId;
-               (messageId, threadId) = sendStatus;
-               io:println("Sent Message Id : " + messageId);
-               io:println("Sent Thread Id : " + threadId);
-           }
-           gmail:GmailError e => io:println(e); //For unsuccessful attempts, returns Gmail Error.
-       }
+## Compatibility
+|                    |    Version     |  
+| :-----------------:|:--------------:| 
+| Ballerina Language | 0.970.0-beta14 |
+|     Gmail API      |    v1         |  
+
+## Sample
+The Gmail connector can be used to send, read, and delete email. First, import the `wso2/gmail` package into the 
```

Comment:
Delete the first sentence: "The Gmail connector can be used to send, read and delete email."

Reply:
Fixed with PR https://github.com/wso2-ballerina/package-gmail/pull/23